docs/DeveloperGuide.md

```diff
@@ -426,29 +424,45 @@ The sequence diagram below illustrates the execution of `find alex`.
 
 ### Edit feature
 
-The edit mechanism implements the following sequence for the method call execute("edit").
+In this section, the functionality of the `edit` feature, the expected execution path, the structure of the **EditCommand** class and the interactions between objects with the **EditCommand** object will be discussed.
 
 #### What is the edit feature
 
 The edit feature allows the user to edit field values of exising student with new values.
 
 The `edit` command is as follows:
 
-`edit 1 n/Poppy p/62536273 ...` where `...` indicates any other additional fields the user wishes to edit.
+`edit [INDEX] [PREFIX/NEW_VALUE]...` where `[PREFIX/NEW_VALUE]...` indicates one or more new fields in which the user wishes to edit.
 
-The original AB3 implementation of this feature allows the same edited value for the corresponding field of the person. `e.g` If a `Person` with `name` of  John (index 1 in the address book) is already present in the address book, then the command `edit 1 n/John` will still work. 
-In addition, for attribute types that need to be unique for each `Person` `e.g.``Phone`, `Email` and `Matriculation Number`, the edited value for these unique attribute types still work even if it already exists in the address book. 
-<br>`e.g.` If there are two `Person`:
-* `name`: John `Email`: john123@gmail.com (Index 1 in the address book)
-* `name`: Johnny `Email` johnny123@gmail.com (Index 2 in the address book)
-<br> The command `edit 2 e/john123@gmail.com` still works and the new `Email` value for Johnny would be updated to `john123@gmail.com` even though this email already exists in the address book for John and is supposed to be unique for each person in the address book. 
+The original AB3 implementation of this feature allows the same edited value for the corresponding field of the person. e.g. If a `Person` with `name` of  John (indexed 1) is already present in the address book, then the command `edit 1 n/John` will still work. 
```

Comment:
this feature allows the same edited value for the corresponding field of the person. 

maybe can say

this feature allows editing fields without making any new changes on the person. 

Can consider.